learn/get-started/work-with-data-using-queries-in-ballerina.md

```diff
@@ -18,42 +18,44 @@ redirect_from:
 
 Ballerina has first-class support for writing SQL-like queries to process data. Language-integrated queries can process any Ballerina iterable.
 
-In this tutorial, you will be writing queries to filter, sort, and join with different data sets and produce new data sets.
-
 ## Set up the prerequisites
 
-To complete this tutorial, you need:
+To run this guide, you need the following prerequisites:
```

Comment:
Is it correct to say run here? How about follow or similar wording?

Reply:
@dakshitha - WDYT?

Reply:
Will be fixed via https://github.com/ballerina-platform/ballerina-dev-website/pull/4696.